opal/mca/threads/qthreads/Makefile.am

```diff
@@ -19,12 +19,41 @@
 # $HEADER$
 #
 
-noinst_LTLIBRARIES = libmca_threads_qthreads.la
+AM_CPPFLAGS = $(opal_qthreads_CPPFLAGS)
 
-libmca_threads_qthreads_la_SOURCES = \
+sources = \
+    threads_qthreads.h \
     threads_qthreads_component.c \
-    threads_qthreads_mutex.c \
     threads_qthreads_condition.c \
-    threads_qthreads_module.c
+    threads_qthreads_module.c \
+    threads_qthreads_mutex.c \
+    threads_qthreads_wait_sync.c \
+    threads_qthreads_mutex.h \
+    threads_qthreads_threads.h \
+    threads_qthreads_tsd.h \
+    threads_qthreads_wait_sync.h
+
+#lib = libmca_threads_qthreads.la
```

Comment:
Nit: no need to commit commented-out code.

Reply:
Thank you for reviewing.  I removed it.